### ISSUE TITLE
maybe permit empty dates in the schema?

### ISSUE THREAD

**Issue by paul-shannon** [davidjamesbeck/eFieldbook#1]
I may be mistaken, but I think the schema requires a valid date in several places.  For the Lushootseed entries, dates are often missing.    I saw this schema definition at [stackoverflow](https://stackoverflow.com/questions/7360906/having-an-empty-xsdate-element-being-validated-ok-by-the-xsd).  maybe worthwhile here?
```
<xs:simpleType name="DateOrEmptyDate">
    <xs:union memberTypes="xs:date">
        <xs:simpleType>
            <xs:restriction base="xs:string">
                <xs:enumeration value=""/>
            </xs:restriction>
        </xs:simpleType>
    </xs:union>
</xs:simpleType>    
```

**Comment by davidjamesbeck** [comment]
Yes, the schema does require valid dates, but your solution looks like a good idea. Especially if we need to extend it to situation where people haven’t been meticulous about their book-keeping.

David

**Comment by paul-shannon** [comment]
Thanks, David.  I’ll fork the repo, make this change later this week, then issue a PR when the time seems right.

eFieldBook has the schema here:

  ./eFieldBook_Qt5/ELFB/ELFB.xsd

Shall I, in my fork, make the small change there?   Or maybe there is a better place?

 - Paul
